### DISCUSSION TITLE
Can I chart WMS with a time component?

### DISCUSSION THREAD

**Discussion by hardreddata** [TerriaJS/terriajs#5452] [Q&A]
Hi community,

I am looking at https://docs.terria.io/guide/connecting-to-data/customizing-data-appearance/feature-info-template/ and https://github.com/TerriaJS/terriajs/blob/master/wwwroot/test/init/charts.json and could use a push. 

Also the power examples from https://raw.githubusercontent.com/TerriaJS/NationalMap-Catalog/master/build/nm_big.json

I have set GeoServer to know about the time dimension following https://docs.geoserver.org/stable/en/user/data/webadmin/layers.html#data-webadmin-layers-edit-dimensions and Terria picks up on this, presenting the time arrows and such on the layer panel on the left.

I cannot seem to get the y values on my chart. I am exploring variations on the below, where time and data are the field names in GeoServer.

```
{
   "catalog":[
      {
         "name":"mylayer",
         "type":"wms",
         "url":"http://mywms/ows",
         "layers":"mylayer",
         "featureInfoTemplate":{
            "template":"<chart x-column='time' y-column='data'></chart>"
         }
      }
   ]
}

```

Any advice is much appreciated.

Thanks.

**Comment by amruthkiran94** [comment]
Hey!
Did you figure out how to go about this? I'm sort of stuck in a similar situation myself.

**Reply by hardreddata** [reply]
I did not. Note that I was using v7 and not v8.
